### PR TITLE
Pass along context in data_spec

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/lib/nl_data_spec.py [hunk 1]
```diff
@@ -83,7 +86,7 @@ def _related_places(dcid):
 def _highlight_svs(sv_df):
   if sv_df.empty:
     return []
-  return sv_df[sv_df['CosineScore'] > 0.4]['SV'].values.tolist()
+  return sv_df[sv_df['CosineScore'] > 0.5]['SV'].values.tolist()
```
Comment (shifucun) [inline server/lib/nl_data_spec.py:89]: @jehangiramjad I need to change this to 0.5 to make "Which cities are the worst in California" work. Currently it detects Count_Person_Urban with score > 0.4

Comment (pradh) [reply server/lib/nl_data_spec.py:89]: We should probably validate it doesn't impact any of the working queries?

Comment (jehangiramjad) [reply server/lib/nl_data_spec.py:89]: I can check that offline and let you know how it looks

**Review by beets — APPROVED** [review]

File: server/lib/nl_data_spec.py (hunk 1 above)
Comment (shifucun) [reply server/lib/nl_data_spec.py:89]: Thanks. If not working, we can revert this back. But maybe tune the model to let this query not returning high score vars.

Comment (jehangiramjad) [reply server/lib/nl_data_spec.py:89]: Sure, sounds good. You can merge then. We can play with it. We an always put this condition on the output of the model's matches, if you want?

**Pull request merged**